spy.go

```diff
@@ -22,17 +81,19 @@ func ParseCommand(cmd string) (string, []string, error) {
 	return stringResult[0], stringResult[1:], nil
 }
 
-// Execute accepts a command argument and writes the result to a writer
-func Execute(w io.Writer, commandLine string) error {
+// Execute accepts a shell command and writes the result to
+// the Session struct writers (Recorder and Terminal)
+func (s *session) Execute(commandLine string) error {
 	name, args, err := ParseCommand(commandLine)
-	fmt.Fprintln(w, commandLine)
+
+	w := io.MultiWriter(s.Terminal, s.Recorder)
```

Comment:
Does it make sense to create this multiwriter in the constructor, instead of here? After all, we'll presumably be using the same multiwriter for every command that we execute; this would save recreating it every time.